### ISSUE TITLE
Duplicate email added to the invitee list

### ISSUE THREAD

**Issue by codejamtc** [appirio-tech/connect-app#3827]
### Expected behavior
User should not be able to add the same email again and again

### Actual behavior
When user add an email and then add the same email with space will remove that email from the textbox, but when user click on 'Send Invite' duplicate email shown on the list

### Steps to reproduce the problem
- Login as Manager
- Click on a project from the project listing page
- Click "Dashboard"
- On "Customer team" > Click 'Manage Invitations'
- Enter 'test2@test.com' and click on the suggestion 
- Enter 'test2@test.com' and add a space after that (Email will be removed, not added to the list
- Do the above steps again
- Now click Send Invite
- Check the list

### Screenshot/screencast

![image](https://user-images.githubusercontent.com/42398485/78197678-aa7e6200-744b-11ea-8f42-66ab81c2549a.png)

https://www.screencast.com/t/5KXSyiV1Z

#### Environment
- OS: Window 10
- Browser (w/version): Chrome 80.0.3987.149 (Official Build) (64-bit)
- User role (client, copilot or manager): Manager
- Account used:  topcoderconnect+Customer@gmail.com

**Comment by vikasrohit** [comment]
@maxceem just wanted to know if it is happening because of email masking.

**Comment by maxceem** [comment]
@vikasrohit I don't think so. It looks like an existent issue that happens due to several edge cases.

**Comment by maxceem** [comment]
This happens the next way:
- client side works incorrectly when we press space, it visually removes email, but it keeps it in the value, which is later is sent to the serveer
- server side we check that we don't create invites for already invited users or members. But we don't check the case when in a single request we are creating several invites for the same user

**Comment by vikasrohit** [comment]
> server side we check that we don't create invites for already invited users or members. But we don't check the case when in a single request we are creating several invites for the same user

Do we create two separate invites in case single request contains same email twice?

**Comment by maxceem** [comment]
> Do we create two separate invites in case single request contains same email twice?

yes

**Comment by maxceem** [comment]
We suppose to fix it on the server-side by filtering members by active when we are retrieving the handles so we don't get duplicates. But for some reason on PROD the Identity Service hangs when we are adding a filter, so we are waiting for the Identity Service on PROD to works correctly first, then we can apply the fix.

The filter works good on DEV:

```js
curl 'https://api.topcoder-dev.com/v3/users?fields=handle,status&filter=email%3Dvikas.agarwal%40topcoder.com%26active%3Dtrue&limit=2' \
  -H 'authority: api.topcoder.com' \
  -H 'pragma: no-cache' \
  -H 'cache-control: no-cache' \
  -H 'accept: application/json, text/plain, */*' \
  -H 'authorization: Bearer <M2M_TOKEN>' \
  -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.122 Safari/537.36' \
  -H 'origin: https://connect.topcoder.com' \
  -H 'sec-fetch-site: same-site' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-dest: empty' \
  -H 'accept-language: en-US,en;q=0.9' \
  --compressed
```

returns only active user

```js
{"success":true,"status":200,"metadata":null,"content":[{"handle":"vikasrohit","status":"A"}]}
```

**Comment by vikasrohit** [comment]
@maxceem not an urgent thing, just landed on this issue while going through unread mails. When you get chance, can you verify if prod seems to be working fine now? If yes, can we close this issue or do we need some changes?

**Comment by maxceem** [comment]
@vikasrohit as a copilot I don't have permission to invite Customer users anymore on PROD.

**Comment by vikasrohit** [comment]
oh, @RishiRajSahu can you verify it?
